<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lookemail</class>
 <widget class="QMainWindow" name="lookemail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>655</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>955</width>
    <height>655</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>955</width>
    <height>655</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#lookmail{
background-color:rgb(255,217,102);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
background-color:rgb(159,197,232);
}</string>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>931</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_subject">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>主    题：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_subject">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>671</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_subject{
border-radius:6px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_recever">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>671</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_recever{
border-radius:6px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#download:hover{
background-color:rgb(204,204,204);
border-radius:10px;
}
#download:!hover{
background-color:rgb(225,225,225);
border-radius:10px;
}
#download:hover:pressed{
background-color:rgb(255,217,102);
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string>下载附件</string>
    </property>
    <property name="icon">
     <iconset resource="Resource.qrc">
      <normaloff>:/new/prefix1/icon/net_32.ico</normaloff>:/new/prefix1/icon/net_32.ico</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_recever">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Agency FB&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_send:hover{
	font: 11pt &quot;新宋体&quot;;
background-color:rgb(204,204,204);
border-radius:10px;
}
#pushButton_send:!hover{
	font: 11pt &quot;新宋体&quot;;
background-color:rgb(225,225,225);
border-radius:10px;
}
#pushButton_send:hover:pressed{
	font: 11pt &quot;新宋体&quot;;
background-color:rgb(255,217,102);
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
    <property name="icon">
     <iconset resource="Resource.qrc">
      <normaloff>:/new/prefix1/icon/send_32.ico</normaloff>:/new/prefix1/icon/send_32.ico</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>609</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_notice">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
